src/libs/NextStepUtils.ts

```diff
@@ -9,9 +10,17 @@ function parseMessage(messages: Message[] | undefined) {
     let nextStepHTML = '';
 
     messages?.forEach((part) => {
+        const isEmail = Str.isValidEmail(part.text);
         // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
-        const tagType = part.type || 'span';
-        nextStepHTML += `<${tagType}>${Str.safeEscape(part.text)}</${tagType}>`;
+        let tagType = part.type || 'span';
```

Comment:
is it possible that `type` will be an empty string? if no I think its safe to use `??` operator

Reply:
@mountiny, can I ask you to check the backend logic for this case. Looks like the app cannot have the empty string case.

Reply:
The `type` key either does not exist or has some value

Reply:
Done - fadfcf4.